src/components/Dialog/ExportSongAudioDialog/Presentation.vue

```diff
@@ -0,0 +1,185 @@
+        >
+          <QToggle v-model="isMono" />
+        </BaseCell>
+        <BaseCell
+          title="音声のサンプリングレート"
+          description="音声のサンプリングレートを変更できます。"
+        >
+          <QSelect
+            v-model="samplingRate"
+            dense
+            name="samplingRate"
+            :options="samplingRateOptions"
+            :optionLabel="renderSamplingRateLabel"
+          >
+          </QSelect>
+        </BaseCell>
+        <BaseCell
+          title="リミッターを適用する"
+          description="ONの場合、0dBを極力超えないように音声を調整します。"
+        >
+          <QToggle v-model="withLimiter" />
+        </BaseCell>
+        <BaseCell
+          title="適用するトラックのパラメーター"
+          description="パン、ボリューム、ミュートのうち、どのパラメーターを書き出し時に適用するか選べます。"
+        >
+          <QOptionGroup
+            v-model="withTrackParameters"
+            type="checkbox"
+            inline
+            :options="trackParameterOptions"
+          />
+        </BaseCell>
+      </QCardSection>
+
+      <QSeparator />
+
+      <QCardActions>
+        <QSpace />
+        <QBtn
+          unelevated
+          align="right"
+          label="キャンセル"
+          color="toolbar-button"
+          textColor="toolbar-button-display"
+          class="text-no-wrap text-bold q-mr-sm"
+          @click="handleCancel"
+        />
+        <QBtn
+          unelevated
+          align="right"
+          label="書き出す"
+          color="toolbar-button"
+          textColor="toolbar-button-display"
+          class="text-no-wrap text-bold q-mr-sm"
+          @click="handleExportTrack"
+        />
+      </QCardActions>
+    </QCard>
+  </QDialog>
+</template>
+
+<script setup lang="ts">
+// NOTE: 前回の設定を引き継ぐため、他のダイアログでやっているようなinitializeValuesはやらない
+import { ref, computed } from "vue";
+import { useDialogPluginComponent } from "quasar";
+import BaseCell from "./BaseCell.vue";
+import { SongExportSetting, TrackParameters } from "@/store/type";
+
+export type ExportTarget = "master" | "stem";
+const { dialogRef, onDialogOK, onDialogCancel } = useDialogPluginComponent();
+
+const modelValue = defineModel<boolean>();
+const emit = defineEmits<{
+  /** 音声をエクスポートするときに呼ばれる */
+  exportAudio: [exportTarget: ExportTarget, setting: SongExportSetting];
+}>();
+
+// 書き出し対象選択
+const exportTargets = [
+  {
+    label: "すべてのトラック",
+    value: "master",
+  },
+  {
+    label: "トラックごと",
+    value: "stem",
+  },
+];
+const exportTarget = ref<ExportTarget>("master");
+
+// モノラル
+const isMono = ref<boolean>(false);
+
+// サンプルレート
+const samplingRate = ref<number>(48000);
+const samplingRateOptions = [24000, 44100, 48000, 88200, 96000];
+const renderSamplingRateLabel = (rate: number) => `${rate} Hz`;
+
+// リミッター
+const withLimiter = ref<boolean>(true);
+
+// パン・ボリューム・ミュート
+const withTrackParameters = ref<(keyof TrackParameters)[]>([
+  "pan",
+  "gain",
+  "soloAndMute",
+]);
+const trackParameterOptions = computed(() => [
+  {
+    label: "パン",
+    value: "pan",
+    disable: !isMono.value,
```

Comment:
ここはこう？
```suggestion
    disable: isMono.value,
```

`disable`でもチェックが付いたままになっているので、パンが無効化されることが少し分かりづらいかもです。チェックを外すようにすると良いかも。（モノラルがオフのときの設定は保持されたほうが良さそう）

Reply:
チェックが外れるようにしました。
トラックのパラメーターをいじってなければ設定が保持されるようになっています。
パンを有効化 -> モノラル有効化 -> 音量とかを切り換え -> モノラル無効化、をするとパンが（触ってないのに）消えますが、まぁそもそもモノラルの切り替えはそんなに起こらないのでパスでいいかなと

Reply:
なるほどです！

確かにモノラルの切り替えはそんなに起こらなそうなので、今の形でも良いと思いますが、
以下の形にしても良いかも。（パンが保持されるようになりますが、煩雑かも…）
```ts
withTrackParametersInner.value = isMono.value
  ? withTrackParametersInner.value.includes("pan")
    ? ["pan", ...value.filter((v) => v !== "pan")]
    : value.filter((v) => v !== "pan")
  : value;
```

Reply:
まぁそこまでこだわるところでもないと思うので今の形で良いと思います。

Reply:
横からすみません！
UI から非表示にしてしまうだけでもいいかもですね！

ただまあ後回しでも確かに良さそう！